setting up alternative keys for nested tmux navigation

sdondley opened the pull request:
probably the simplest solution

christoomey commented:
Apologies if I misunderstood, but with #336 merged, this one now has a conflict. Mind taking another look here, @sdondley?

sdondley commented:
Yeah I was afraid of that. I changed the first line of the following paragraph which was also changed in previous patch. I can go back and redo it unless you want to fix it. On Jan 8, 2023, at 2:49 PM, Chris Toomey ***@***.***> wrote:﻿
Apologies if I misunderstood, but with #336 merged, this one now has a conflict. Mind taking another look here, @sdondley?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

sdondley commented:
Closing this in favor of new one.
